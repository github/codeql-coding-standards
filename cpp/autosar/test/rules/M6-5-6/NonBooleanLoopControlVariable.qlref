rules/M6-5-6/NonBooleanLoopControlVariable.ql