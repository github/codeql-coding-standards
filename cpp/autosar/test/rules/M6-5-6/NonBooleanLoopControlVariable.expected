| test.cpp:21:5:21:5 | y | Loop control variable y is modified in a statement and must be of type bool. |
| test.cpp:35:32:35:32 | z | Loop control variable z is modified in a statement and must be of type bool. |
| test.cpp:36:5:36:5 | z | Loop control variable z is modified in a statement and must be of type bool. |
